### ISSUE TITLE
i get all the libs needed,but the parser only make a ast with out class code block

### ISSUE THREAD

**Issue by onewaymyway** [teotigraphix/as3-commons-asblocks#5]
it's ok now,eventhough i don't know why,this project is amazing!thank you for your great work!!!

**Comment by onewaymyway** [comment]
i find another bug.
it can't work with the for expression like this
            var i:int;
            var len:int;
            len=2;
            for(i=0 ; i<len ; i++ )
            {
                trace("forTrace:"+1);
            }

but work with this
            var i:int;
            var len:int;
            len=2;
            for(i=0 ;( i<len) ; i++ )
            {
                trace("forTrace:"+1);
            }

will you fix this?or wil you tell me where shall i hack so i can fix this

**Comment by onewaymyway** [comment]
it works well with this too. i was confused
            var i:int;
            var len:int;
            len=2;
            for(i=0 ; i<=len ; i++ )
            {
                trace("forTrace:"+1);
            }

**Comment by teotigraphix** [comment]
Yeah I don't really offer support on this, I kept the frameworks up more for old portfolio stuff. :)
